client/src/pages/Dashboard.jsx

```diff
@@ -1,31 +1,58 @@
 import React from "react";
+import axios from "axios";
+import {Redirect, useHistory } from "react-router-dom";
 import Button from "@material-ui/core/Button";
 import CssBaseline from "@material-ui/core/CssBaseline";
 import TextField from "@material-ui/core/TextField";
 import Paper from "@material-ui/core/Paper";
-import { useHistory } from "react-router-dom";
 
 export default function Dashboard() {
   const history = useHistory();
+  const [user, setUser] = React.useState(null)
+  const [loading, setLoading] = React.useState(true)
+
+  const getUser = async () => {
+    try {
+      const {data} = await axios.get('/users/auth');
```

Comment:
We don't want to be making the same API call from multiple places on the front end, though you can leave fixing that to the refactor PR